packages/form_bloc/test/form_bloc/form_bloc_utils_test.dart

```diff
@@ -726,23 +726,21 @@ void main() {
 
     */
     // TODO: Add more tests
-    group('fieldBlocsToFieldBlocsStates', () {
```

Comment:
I commented on the print tests simply because they weren't testing anything

PS: The current (and previous) implementation regarding reading blocs or values in state is not consistent.
       The status of the form bloc stores the statuses internally and if a fieldbloc is updated the saved status is not updated accordingly